src/toil/fileStore.py

```diff
@@ -1603,7 +1607,8 @@ def open(self, job):
                                            disk=diskUsed,
                                            humanRequestedDisk=bytes2human(jobReqs),
                                            requestedDisk=jobReqs))
-            self.logToMaster(logString, level=logging.INFO)
+            self.logToMaster(logString, level=(logging.DEBUG if self.jobName.startswith(self.debugJobNames)
```

Comment:
Thanks for looking at the PR. What I'm trying to do with the different levels is move the internal events (in CWL, things like workflow and job invocations) to the DEBUG log, while leaving the higher level outputs (actual commands being run) in the INFO output. This is a super useful output to have for tasks since it lets you measure progress of a workflow but gets overwhelmed by all the internal messages. The WARNING below is also useful but a separate thing you'd want reported. Hopefully that helps explain the logic behind having two different levels here.